src/static/css/page.css

```diff
@@ -187,7 +187,7 @@
 }
 
 .content-banner {
-  max-width: 100%;
+  width: 100%;
```

Comment:
Unrelated change but notice the banner didn't go full width on very wide screens, and that looked weird no we are right aligning the chapter dates.